3.analysis/scripts/nbconverted/3.t_tests.py

```diff
@@ -199,7 +199,7 @@
 
 
 # Combining and saving t test results
-t_test_results_df = pd.concat([t_test_adult_vs_ped, t_test_male_vs_female, t_test_adult_male_vs_ped_male, t_test_adult_female_vs_ped_female, t_test_ped_male_vs_ped_female, t_test_adult_male_vs_adult_female])
+t_test_results_df = pd.concat([t_test_adult_vs_ped, t_test_male_vs_female, t_test_adult_male_vs_ped_male, t_test_adult_female_vs_ped_female, t_test_ped_male_vs_ped_female, t_test_adult_male_vs_adult_female]).reset_index(drop=True)
```

Comment:
One comment, one question here:

Comment: consider formatting code to span multiple lines for ease of reading i.e.

t_test_adult_vs_ped, 
t_test_male_vs_female, 
t_test_adult_male_vs_ped_male,

You might consider talking with Erik about using a pre-commit hook for a code formatted such as [black](https://black.readthedocs.io/en/stable/)

Question:
Why t-tests across multiple groups? Have you considered also doing an ANOVA and post hoc test? There may exist multiple correct answers, consider adding comments as to why a t-test or ANOVA.

Reply:
I initially threw together the t-tests across groups since it was quick and adds further analysis. The most important t-tests were adult vs pediatric and male vs female. In the near future I am planning on switching over to ANOVA.